dashboard/src/actions/repos.test.tsx

```diff
@@ -339,6 +339,41 @@ describe("fetchRepos", () => {
     await store.dispatch(repoActions.fetchRepos(namespace, "other-ns"));
     expect(store.getActions()).toEqual(expectedActions);
   });
+
+  it("fetches repos only if the namespace is the one used for global repos", async () => {
+    AppRepository.list = jest
+      .fn()
+      .mockImplementationOnce(() => {
+        return { items: [{ name: "repo1", metadata: { uid: "123" } }] };
+      })
+      .mockImplementationOnce(() => {
+        // will be skipped
+        return {
+          items: [
+            { name: "repo1", metadata: { uid: "321" } },
+            { name: "repo2", metadata: { uid: "123" } }, // same uid
```

Comment:
you mean `// different uid`. In any case, I'd avoid inline comments if they are not super necessary, usually it only makes sense to the one writing them :)